package.json

```diff
@@ -18,23 +28,32 @@
     "email": "support@digitalbazaar.com",
     "url": "https://digitalbazaar.com"
   },
+  "engines": {
+    "node": ">=14"
+  },
   "bugs": {
     "url": "https://github.com/digitalbazaar/mocha-w3c-interop-reporter/issues"
   },
   "homepage": "https://github.com/digitalbazaar/mocha-w3c-interop-reporter#readme",
   "dependencies": {
+    "app-root-path": "^3.0.0",
     "chalk": "^4.1.0",
-    "esm": "^3.2.25",
     "handlebars": "^4.7.7",
     "mocha": "^8.3.2",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "eslint": "^7.24.0",
-    "eslint-config-digitalbazaar": "^2.6.1",
-    "eslint-plugin-jsdoc": "^32.3.0"
+    "eslint": "^8.15.0",
+    "eslint-config-digitalbazaar": "^3.0.0",
+    "eslint-plugin-jsdoc": "^39.2.9",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.72.1"
   },
   "peerDependencies": {
     "mocha": ">= 6.0"
-  }
+  },
+  "files": [
```

Comment:
To make it easier to edit, nice if this is up below "exports".

Reply:
Addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/ef10a5e7ae48da4199f898ba6053298bd6399530